frontend/src/components/DropdownListItems.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Button, Dropdown, Menu} from "antd";
+import {CloseOutlined, LineOutlined, MenuOutlined} from "@ant-design/icons";
+
+const DropdownListItems = ({ListItems}) => {
+  const menuListItems = (
+    <Menu>
+      {ListItems ? ListItems.map((item, i) => <Menu.Item key={i.toString()}>{item}</Menu.Item>) : <Menu.Item/>}
+    </Menu>
+  );
+
+  return (
+    ListItems && ListItems.length > 1 ? 
+    <Dropdown overlay={menuListItems} placement="bottomRight">
+      <div>
+        <MenuOutlined /> {ListItems && ListItems[0]}
```

Comment:
If there are many sequences, maybe it would be better to show a "Expand sequences" message, instead of the first item of the list i.e. ListItems[0]

Reply:
expand sequences is no information at all the first one is still some additional info ... I do not know really. The lines makes it already visible that it is a list... Not sure anything would be gained by not showing the first one.

Reply:
We can at least add a way to tell the user that they need to hover on it to be able to see the list. Like we did with the sampleQC flag in SampleList.